tests/migrations/failure.py

```diff
@@ -0,0 +1,20 @@
+from migration_steps import MigrationStep
+
+# If this filename is not 0.0.1.py, then fail already during validation
+FAIL_AT_VALIDATE = False if "0.0.1.py" in __file__ else True
```

Comment:
This feels very magical to me. Why not have a separate validation-failure.py?

(But if we keep it, `"0.0.1.py" not in __file__`)

Reply:
Organic growth leftover! Created a new file specific to the test.